sklearn/multioutput.py

```diff
@@ -37,12 +39,90 @@ def _fit_estimator(estimator, X, y, sample_weight=None):
     return estimator
 
 
+def _partial_fit_estimator(estimator, X, y, classes=None, sample_weight=None,
+                           first_time=True):
+    if first_time:
+        estimator = clone(estimator)
+    else:
+        estimator = copy.copy(estimator)
+
+    if sample_weight is not None:
```

Comment:
`classes` is required for classifiers' `partial_fit`, but I think we have no requirement that `sample_weight` be supported.